test/functional/comment_controller_test.rb

```diff
@@ -42,6 +42,29 @@ def teardown
     assert_template partial: 'questions/_comment'
   end
 
+  test 'should create wiki comments' do
+    UserSession.create(rusers(:bob))
+    assert_difference 'Comment.count' do
```

Comment:
@jywarren , ah yes verifying the added comment is an one attached to a wiki is necessary. I'm adding it now. 

And as per unit tests, yes I get it, will add unit tests to create comments for the wiki. Will think more of necessary unit tests and will update the orginial checklist(https://github.com/publiclab/plots2/issues/1438)